arviz/tests/external_tests/test_data_pymc.py

```diff
@@ -434,3 +435,58 @@ def test_save_warmup(self, save_warmup):
         if save_warmup:
             assert idata.warmup_posterior.dims["chain"] == 2
             assert idata.warmup_posterior.dims["draw"] == 100
+
+    @pytest.mark.skipif(
+        hasattr(pm.backends.base.SamplerReport, "n_draws"), reason="requires pymc3 3.8 or lower",
+    )
+    def test_save_warmup_issue_1208_before_3_9(self):
+        with pm.Model():
+            pm.Uniform("u1")
+            pm.Normal("n1")
+            trace = pm.sample(
+                tune=100,
+                draws=200,
+                chains=2,
+                cores=1,
+                step=pm.Metropolis(),
+                discard_tuned_samples=False,
+            )
+            assert isinstance(trace, pm.backends.base.MultiTrace)
+            assert len(trace) == 300
+
+            # <=3.8 did not track n_draws in the sampler report,
+            # making from_pymc3 fall back to len(trace) and triggering a warning
+            with pytest.warns(UserWarning):
```

Comment:
can you use `match`? something like `match="Warmup samples"` should be enough, it's only to prevent the test from passing if there is some unrelated warning.

Reply:
didn't know that feature, thanks!